.circleci/config.yml

```diff
@@ -167,13 +166,15 @@ jobs:
             cp -f /usr/local/asdf/tool-versions-copy .tool-versions
             mix deps.get || mix deps.get
             mix deps.get
-            git --no-pager diff
-            git -c user.name='AntikytheraDeploy' -c user.email='antikythera-gr@access-company.com' commit -a -m '[CI] Update antikythera'
+            if [ -n "$(git --no-pager diff)" ]; then
```

Comment:
The current code is fine, but I prefer to use `if git diff --quiet; then`.
https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---quiet

Reply:
Thank you!
I fixed in https://github.com/access-company/antikythera/pull/82/commits/ecac1c0ff84ccfc0880ac710feba1b461efcf3a7